README.md

```diff
@@ -29,6 +29,15 @@ Run `./build.sh`* to build. The first time you run this, it could take a while,
 
 \* On Windows, it's `.\build.ps1`. Though you may need `powershell.exe -executionpolicy bypass -file .\build.ps1` due to permissions issues. I don't personally know a thing about Windows permissions, so if anyone has a better idea please let me know.
 
+## Cross-compiling
+
+If we're on x86-64 Linux, we can use `nix develop` to enter a shell from which we can cross-compile:
+```sh
+~/.ghcup/bin/runghc Build.hs --target=x86_64-w64-mingw32
+~/.ghcup/bin/runghc Build.hs --target=x86_64-unknown-linux-musl
+~/.ghcup/bin/runghc Build.hs --target=aarch64-unknown-linux-gnu
+```
+
```

Comment:
We could do with a section on deploying the resulting artefacts.

Reply:
Ideally we'll manage to fully statically link everything, meaning we'll just have a single executable file to deploy.

If we are stuck with shared libraries, we'll need commands like:
- `rsync result/ ~Desktop/monpad-windows-nix -a --copy-links -v`
- `TARGET_NAME=... nix copy --to ssh://$TARGET_NAME .#aarch64-unknown-linux-gnu:monpad:exe:monpad && scp ./result/bin/monpad $TARGET_NAME:tmp/hello-cross-nix`

Reply:
We now have static Linux builds, but Windows needs a load of DLLs (weirdly this is an inversion of the usual non-Nix state of things). Can we compile statically with MinGW?